FileSystem.py

```diff
@@ -0,0 +1,19 @@
+#File System
+#Write a program that accepts a file path and returns the number of words in the file.
+def getWords(filePath):
+    #Assumtpion that every word ends with a space or one of .!?
+    with open(filePath,"r") as f:
```

Comment:
You could have used readlines and split using the space character for a more pythonic implementation